docs/int/openapi.yaml

```diff
@@ -1148,6 +1354,33 @@ components:
         name:
           type: string
 
+    NamedReferenceResponse:
```

Comment:
This should be 
- [ ] moved to the /components/responses/ section
- [ ] remove 'Response' from the schema name